docs/glossary.md

```diff
@@ -52,10 +58,14 @@ title: Glossary
 
 **[Nyx](https://nymtech.net/docs/stable/integrations/payment-integration-overview/)** - A CosmWasm-enabled blockchain smart contracts platform that functions as the backbone of the Nym network, used to keep track and provide the history of the NYM token's transactions.
 
+**[Packets](https://cypherpunks.ca/~iang/pubs/Sphinx_Oakland09.pdf)** - A collection of data that can be used by computers which need to communicate with each other, usually as part of a network in the packet format of sphinx.
```

Comment:
Are we defining packets in general, or our packets? If we're defining them in general then do that, and add a sentence at the end saying what our packets are. Tbh I don't think we need to do that, just defining Sphinx as the packet format we use might be the better way to go.